packages/react-html/tsconfig.json

```diff
@@ -1,12 +1,10 @@
 {
-  "extends": "../../tsconfig.json",
-  "compileOnSave": false,
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "outDir": "./dist",
     "baseUrl": "./src",
     "rootDir": "./src"
   },
-  "include": [
-    "src/**/*.ts"
-  ]
+  "include": ["src/**/*.ts", "src/**/*.tsx"],
+  "references": [{"path": "../react-serialize"}]
```

Comment:
Adding these lets `tsc -b` understand how to traverse our in-repo dependencies and build them in the right order, and lets you build from within the package's sub-folder and still rebuild projects it depends on